tools/ci_build/github/azure-pipelines/linux-migraphx-ci-pipeline.yml

```diff
@@ -0,0 +1,54 @@
+trigger: none
+
+name: 'linux_ci_$(Date:yyyyMMdd)_$(Rev:r)'
+jobs:
+- job: AMDMIGraphX_CI
+  pool: 'AMD-GPU'
```

Comment:
Unless you can have new machines for this pool, you should let the existing users of this pool know that half of their resource will be allocated to this new pipeline.

Reply:
This pipeline takes nearly half an hour, less than ROCm CI pipeline(1h 20 min). We have 8 available agents in `AMD  pool` and  only turn on 4 agents for ROCm CI pipeline now. We can turn on more agent if Migraphx pipeline causes a shortage of resources.